maintenance/projects/js-toolkit/packages/liferay-npm-bundler/src/jar/index.js

```diff
@@ -300,3 +302,18 @@ function getSystemConfigurationJson() {
 
 	return configurationJson.system;
 }
+
+/**
+ * Get the OSGi bundle version from the package.json version. Respects
+ * the different types of version classifiers.
+ * @return {object}
+ */
+function getBundleVersionAndClassifier(pkgJsonVersion) {
+	const parts = pkgJsonVersion.split('-');
+	if (parts.length > 1) {
+		return parts[0] + '.' + parts.slice(1).join('-');
+	}
+	else {
+		return pkgJsonVersion;
+	}
+}
```

Comment:
So this is a near duplicate of [these lines](https://github.com/liferay/liferay-frontend-projects/blob/fc0a3cb5f972758db6277eb1c876a20a2e7b01b7/maintenance/projects/js-toolkit/packages/liferay-npm-bundler/src/jar/manifest.ts#L33-L39), so we should move this function to somewhere common where the same logic can be used by both places. The other instance of this already has test coverage too, as a bonus. Requesting changes @aschaeffer to get your thoughts on that.